manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0
+Installers:
+- Architecture: neutral

```

Comment:
```suggestion
- Architecture: x64
```
Neutral means the installer supports all architectures (even the arm ones I believe). Looking at the installer URL and the architecture from Task Manager, I don't think this is the case.
![image](https://user-images.githubusercontent.com/88161975/232231264-caae4a33-a2e0-4be2-a324-abb8731f881d.png)